app/models/customer.rb

```diff
@@ -16,7 +16,7 @@ class Customer < ApplicationRecord
 
   belongs_to :enterprise
   belongs_to :user, class_name: "Spree::User", optional: true
-  has_many :orders, class_name: "Spree::Order"
+  has_many :orders, class_name: "Spree::Order", dependent: :destroy
```

Comment:
Hmm this doesn't look right: I would think that we don't want to be destroying orders.

I had a quick look around and noticed at the bottom of this file the intention is to handle this situation differently:
https://github.com/openfoodfoundation/openfoodnetwork/blob/master/app/models/customer.rb#L73

So we want orders to stay when a customer is destroyed. Rails doesn't seem to have an explicit option like `dependent: :keep`, so I guess we need to disable the rubocop here. It can be done with an inline comment, I think like this from memory:

```suggestion
  has_many :orders, class_name: "Spree::Order" # rubocop:disable Rails/HasManyOrHasOneDependent
```

Reply:
Ok I now realise that `dependent: :nullify` is what we want here; that's what the code I linked to is doing already.

Reply:
In order to use the default, which is what was there before, you can just add `dependent: nil`. That is always an option. I just got a little aggressive with db cleaning I guess.